src/pages/homepage.tsx

```diff
@@ -93,117 +84,140 @@ const HomePage = () => {
+        <Container gridCols="9" gridGutters="8" flex={true}>
+          <Column width={40} alignX="center" alignY="center">
+            <IconTwitter size="128" color="var(--colorBlack400)" strokeWidth="1px" />
+          </Column>
+          <Column width={60} subtitle={t('homepage:follow-us.title')}>
+            <TextContent marginTop={false}>
+              <p className="mt__M">{t('homepage:follow-us.desc')}</p>
               <Link href="https://twitter.com/OpenTerms">
-                <Button>{t('homepage:cta_public.case_studies.button.label')}</Button>
+                <a target="_blank" rel="noopener">
+                  <Button className="mb__0">{t('homepage:follow-us.button.label')}</Button>
+                </a>
               </Link>
-              <LinkIcon iconColor="var(--colorBlack400)" href="/case-studies" small={true}>
-                {t('homepage:cta_public.case_studies.link.label')}
-              </LinkIcon>
-            </ButtonBlock>
-            <ButtonBlock
-              title={t('homepage:cta_public.subscribe.title')}
-              iconName="FiMail"
-              iconColor="var(--colorBlack400)"
+            </TextContent>
+          </Column>
+        </Container>
+      </Container>
+
+      {/* Reuses */}
+      <Container gray={true} layout="wide" paddingX={false}>
+        <Container gridCols="12" gridGutters="11">
+          <CardList title={t('homepage:reuses.title')} subtitle={t('homepage:reuses.subtitle')}>
+            <Card
+              image={`/images/reuses/memos-elections-${router?.locale}.png`}
+              className="text__center"
+              title={t('homepage:reuses.memos-elections-fr.title')}
+              subtitle={t('homepage:reuses.memos-elections-fr.subtitle')}
+              author={t('homepage:reuses.memos-elections-fr.author')}
+              link="https://www.reset.tech/resources/memos-on-platforms-behaviour-during-the-2022-french-elections/"
+              center={true}
               white={true}
-            >
-              <Link href="/subscribe">
-                <Button>{t('homepage:cta_public.subscribe.button.label')}</Button>
-              </Link>
-              <LinkIcon
-                iconColor="var(--colorBlack400)"
-                href="https://github.com/ambanum/OpenTermsArchive#by-rss"
-                small={true}
-              >
-                {t('homepage:cta_public.subscribe.link.label')}
-              </LinkIcon>
-            </ButtonBlock>
-            <ButtonBlock
-              title={t('homepage:cta_public.explore.title')}
-              iconName="FiGithub"
-              iconColor="var(--colorBlack400)"
+            />
+            <Card
+              image="/images/reuses/tosdr.jpg"
+              title={t('homepage:reuses.tosdr.title')}
+              subtitle={t('homepage:reuses.tosdr.subtitle')}
+              author={t('homepage:reuses.tosdr.author')}
+              link="https://tosdr.org"
+              center={true}
```

Comment:
For the record, you don't need the `={true}`

Reply:
`center` Card prop is false by default 👀 

```typescript
const Card: React.FC<CardProps> = ({
  children,
  center = false,
  ...props
})
```

Reply:
I mean, you can do
```
author={t('homepage:reuses.tosdr.author')}
link="https://tosdr.org"
center
```